devtools/docker.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+DIR="$(cd "$(dirname "$0")/.." && pwd)"
```

Comment:
Usually, we put a description of the script here and in devtools/README.

It would be nice to follow this :D

Reply:
OK, done.